app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListAdapter.java

```diff
@@ -66,4 +71,16 @@ public void bindView(View view, Context context, Cursor cursor) {
                 break;
         }
     }
+
+    private void delete() {
```

Comment:
Explore media wiki API for deletion to check if it could serve our purpose. https://www.mediawiki.org/wiki/API:Delete

Reply:
ok, Sure sir.